python/fnllm/fnllm/openai/llm/chat_text.py

```diff
@@ -125,14 +127,15 @@ async def _execute_llm(
     ) -> OpenAIChatOutput:
         name = kwargs.get("name")
         history = kwargs.get("history", [])
+        nocache = kwargs.get("nocache", False)
         local_model_parameters = kwargs.get("model_parameters")
         messages, prompt_message = build_chat_messages(prompt, history)
         completion_parameters = self._build_completion_parameters(
             local_model_parameters
         )
 
         completion = await self._call_completion_or_cache(
-            name, messages=messages, parameters=completion_parameters
+            name, messages=messages, parameters=completion_parameters, bypass=nocache
```

Comment:
same as bypass, what are we bypassing? It is not explicit enough

Reply:
Sure, changing to bypass_cache